build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -56,6 +62,8 @@
                             SetupRestoreTestsInputs;"
           Inputs="@(RestoreTestsInputs)"
           Outputs="@(RestoreTestsInputs->'%(RelativeDir)project.lock.json')">
+    <CallTarget Targets="CleanTestProjectsLockFiles"/>
```

Comment:
The calls are disperate. Each Clean target is tied to a specific set of lock files. e.g. CleanTestProjectLockFiles vs. CleanTestAssetProjectLockFiles.